README.md

```diff
@@ -415,6 +415,85 @@ Note: the `path` will be blank when using a `Files` type scan and it will
 use the same logic as `JSONData` when the response's content type is
 `application/json` (i.e. it will be the path down the traversed keys).
 
+### Container Image
+
+This allows you to pull a remote container image to scan. It unpacks and scans
+the Image, Config and Manifest.
+
+#### Request
+
+```json
+{
+  "id": "1c7387179582ae1e9bc23123a10bddc6317fe6a5362efd2ae4019e34cccd8420",
+  "kind": "ContainerImage",
+  "resource": "quay.io/wizzy/fake-leaks:v1.0.2"
```

Comment:
If you want, we can throw a container file in `fake-leaks` that just slurps up everything and publish under `quay.io/leaktk/fake-leaks:latest`.

Not a blocker for this MR. Just a note.